src/connectionconfig/connectionDialogWebViewController.ts

```diff
@@ -314,38 +293,129 @@ export class ConnectionDialogWebViewController extends ReactWebViewPanelControll
 						validationMessage: ''
 					};
 				}
-			},
-			{
-				propertyName: 'database',
-				label: 'Database',
-				required: false,
-				type: FormComponentType.Input,
-			},
-			{
-				propertyName: 'trustServerCertificate',
-				label: 'Trust Server Certificate',
-				required: false,
-				type: FormComponentType.Checkbox,
-			},
+			}
+		});
+
+		components.set('profileName', {
+			option: undefined,
+			component: {
+			propertyName: 'profileName',
+			label: 'Profile Name',
```

Comment:
should these labels be localized?

Reply:
I've localized the ones in the controller for now, but Aasim's working on a larger localization effort (in `aasim/feat/newloc`) that'll enable localization in the webviews as well, as that isn't possible with the current system.